File: 3736-MinimumMovesToEqualArrayElementsIii/3736-MinimumMovesToEqualArrayElementsIii.java
// Last updated: 9/12/2026, 7:18:35 AM
class Solution {
    public int minMoves(int[] nums) {
        Arrays.sort(nums);
        int sum=0;
        for(int i=0;i<nums.length;i++){
            int a=nums[nums.length-1]-nums[i];
            sum=sum+a;
        }
        return sum;
    }
}

File: 3736-MinimumMovesToEqualArrayElementsIii/3736-MinimumMovesToEqualArrayElementsIii.py
# Last updated: 9/12/2026, 7:18:29 AM
class Solution(object):
    def minMoves(self, nums):
        nums.sort()
        sum1=0
        for i in range(0,len(nums)):
            a=nums[len(nums)-1]-nums[i]
            sum1+=a
        return sum1